### DISCUSSION TITLE
Fitbit now requires https

### DISCUSSION THREAD

**Discussion by psonnera** [NightscoutFoundation/xDrip#2244] [General]
[Edit] New Fitbit app 3.63.1 requires https.
This is killing Glance and Sentinel watchfaces using web server that is only http.
3.58 seems ok.

**Comment by tzachi-dar** [comment]
Are you sure? I have been using a fitbit with the same sw version and with sential and it seems to work well (on versa 3).

**Reply by jiresongit** [reply]
3.64.5 broke it

**Reply by Cheun1** [reply]
All is ok with nightscout and dexcom , only pb with xdrip web server , when i switch to an old version of fitbit app (3.62  all is ok ). And i block update on play store account .

**Comment by Cheun1** [comment]
3.63.1 also.

**Comment by jamorham** [comment]
As far as I know when this happened with garmin, complaints were made to them that 127.0.0.1 address shouldn't require https and they added an exception to the system to support it.

**Reply by psonnera** [reply]
@Rytiggy showed me it's what's supposed to happen but doesn't
![image](https://user-images.githubusercontent.com/33809361/181525276-6bbb239e-3e95-48ea-9a21-c4e3b8dac063.png)

**Reply by jamorham** [reply]
So it sounds like the fitbit app authors have introduced a regression with the most recent version so they should have this issue raised with them

**Comment by tzachi-dar** [comment]
Not realy related to this thread, but it seems that our fitbit (versa3) keeps disconnecting every few hours.
Does it work well for people? Anything that should be done to prevent the disconnections?

**Reply by psonnera** [reply]
All I know about it is battery savings settings in Android...

**Reply by Cheun1** [reply]
You have to take a step back because 3.65 is only valid for os10 and higher, which is a shame and not deployed in all countries, it is also certain that given the evolution of the Glance and Sentinel applications, the versa, versa lite will have struggling to work, only the versa 3 and Sense with 128 of memory against the other 64 are successful.

**Comment by psonnera** [comment]
It looks like 3.65 fixed the issue.
